pkg/imagedistributor/interface.go

```diff
@@ -22,7 +22,7 @@ import (
 
 type Distributor interface {
 	// Distribute each files under mounted cluster image directory to target hosts.
-	Distribute(hosts []net.IP, dest string) error
+	Distribute() error
```

Comment:
Does this change apply to the other two methods as well？

Reply:
not changes, hosts IP is from ClusterImageMountInfo.